.gitmodules

```diff
@@ -1,9 +1,9 @@
 [submodule "bids-examples"]
-	path = bids-examples
+	path = tests/bids-examples
```

Comment:
why not `tests/data`? I would say that's where this fits better, no?

Reply:
Fair point - I thought about this too, but I'm not sure if I 100% agree... `tests/data` includes files that we maintain purely for testing the CLI, and every file in there is used in the test suite (and should be documented in `test/data/README.md`).

In contrast, we only use a few files from `bids-examples` and `neurobagel-examples`, and since they are external repos, I think it makes sense to keep them separate from the static TSV & JSON test input files.

I think an option that might make more sense is something like:

```bash
tests/
   data/
     bids-examples/
     neurobagel_examples/
     tsv_json_datasets/  # or similar
   ...
```

with everything currently in `tests/data` moved to `tsv_json_datasets`. But I'm honestly not sure that the extra layer of abstraction is needed or that much better than what we currently have, since we already have fixtures for each of these paths anyhow. 🤷

From a visibility / submodule tracking standpoint, I'm also a little hesitant to have the submodules buried too deep in the directory tree 😅

For now, I've added this as a checkbox to consider in our other existing issue for reorganizing the actual test data: https://github.com/neurobagel/bagel-cli/issues/72.